spec/features/personas/sign_in_as_persona_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 feature "Sign In as Persona" do
   around do |example|
-    Capybara.app_host = "https://#{ENV["PLACEMENTS_HOST"]}"
+    Capybara.app_host = "https://#{ENV["CLAIMS_HOST"]}"
```

Comment:
Why is this being changed?

Reply:
This is a good question! The default user is a claims user, so all the tests fail unless we are in the claims context (or change all the users to be placement users.

In the fuller PR (for the support_user journey), I've split this into two different tests. Personal logins for each service, as they will diverge as the user journeys become more specific to the service context. Happy to walk through it if that doesn't make sense.